views/frontend/blog.php

```diff
@@ -1,30 +1,29 @@
 <h2>TOUS LES ARTICLES</h2>
 
 <?php
-    $posts = $model_article->get_posts_blog_all();
+$count = 0; // Nécessaire pour pouvoir beaucoup sur les 2 tableaux $posts et $admins en même temps.
     foreach($posts as $post){
 ?>
-
     <div class="row">
         <div class="col s12 m12 l12">
-            <h1><?= $post->title ?></h1>
-            <h5 class="grey-text">Le <?= date("d/m/Y à H:i",strtotime($post->date)); ?> par <?= $post->name ?></h5>
+            <h1><?= $post->getTitle() ?></h1>
+            <h5 class="grey-text">Le <?= date("d/m/Y à H:i",strtotime($post->getDate())); ?> par <?= $admins[$count]->getName() ?></h5>
             <div class="row">
                 <div class="col s12 ">
-                    <?= substr(nl2br($post->chapo),0,1200) ?>...
+                    <?= substr(nl2br($post->getChapo()),0,1200) ?>...
                 </div>
                 <div class="row"></div>
                 <div class="col s12 m6 l8">
-                    <?= substr(nl2br($post->content),0,1200) ?>...
+                    <?= substr(nl2br($post->getContent()),0,1200) ?>...
                 </div>
                 <div class="col s12 m6 l4">
-                    <img src="public/img/posts/<?= $post->image ?>" class="materialboxed responsive-img" alt="<?= $post->title ?>"/>
+                    <img src="public/img/posts/<?= $post->getImage() ?>" class="materialboxed responsive-img" alt="<?= $post->getTitle() ?>"/>
                     <br/><br/>
-                    <a class="btn light-blue waves-effect waves-light center" href="/article/<?= transforme_en_url($post->title) ?>-<?= $post->id ?>">Lire l'article complet</a>
+                    <a class="btn light-blue waves-effect waves-light center" href="/article/<?= transforme_en_url($post->getTitle()) ?>-<?= $post->getId() ?>">Lire l'article complet</a>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [All output should be run through an escaping function (see the Security sections in the WordPress Developer Handbooks), found '$post'.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)